functional/FunctionalTest.java

```diff
@@ -4182,33 +4189,9 @@ public static void listenBucketNotification_test1() throws Exception {
         }
       }
 
-      mintSuccessLog(
-          "listenBucketNotification(String bucketName, String prefix, "
-              + "String suffix, String[] events)",
-          null,
-          startTime);
+      mintSuccessLog(methodName, null, startTime);
```

Comment:
the `prefix`, `suffix` and `events` can be logged as `args`

Reply:
Done